@hipstersmoothie mind fixing the linting error?

> /tmp/storybook/app/preact/src/client/preview/render.tsx
  3:1  error  'common-tags' should be listed in the project's dependencies. Run 'npm i -S common-tags' to add it  import/no-extraneous-dependencies

--- a/app/html/package.json
+++ b/app/html/package.json
@@ -34,6 +34,7 @@
   "dependencies": {
     "@storybook/addons": "5.3.0-alpha.2",
     "@storybook/core": "5.3.0-alpha.2",
+    "@types/webpack-env": "^1.13.9",
     "common-tags": "^1.8.0",
     "core-js": "^3.0.1",
     "global": "^4.3.2",

--- a/app/preact/package.json
+++ b/app/preact/package.json
@@ -36,6 +36,7 @@
     "@storybook/addons": "5.3.0-alpha.2",
     "@storybook/core": "5.3.0-alpha.2",
     "common-tags": "^1.8.0",
+    "@types/webpack-env": "^1.13.9",
     "core-js": "^3.0.1",
     "global": "^4.3.2",
     "regenerator-runtime": "^0.12.1"

--- a/app/vue/package.json
+++ b/app/vue/package.json
@@ -34,6 +34,7 @@
   "dependencies": {
     "@storybook/addons": "5.3.0-alpha.2",
     "@storybook/core": "5.3.0-alpha.2",
+    "@types/webpack-env": "^1.13.9",
     "common-tags": "^1.8.0",
     "core-js": "^3.0.1",
     "global": "^4.3.2",

